jaeger/charts/jaeger/templates/rbac.yaml

```diff
@@ -0,0 +1,26 @@
+---
+###
+### collector RBAC
+###
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: collector
+  namespace: {{.Values.namespace}}
+  labels:
```

Comment:
what is the purpose of these labels?

Reply:
These are not helpful for `serviceAccounts`, but I kept them as we had them for services and deployments.

Removing them.